.env

```diff
@@ -0,0 +1,2 @@
+api = https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=04c35731a5ee918f014970082a0088b1&page=
+searchapi = https://api.themoviedb.org/3/search/movie?&api_key=04c35731a5ee918f014970082a0088b1&query=
```

Comment:
Use upper case on environment's variable names. Aka: 
THE_MOVIE_API_HOST=https://api.themoviedb.org/3

You don't need 'searchapi' here. Just get the host and hardcode the routes into the HTTP calls.